src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -695,7 +695,8 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter)
   def verbosity() = {
     val old = intp.printResults
     intp.printResults = !old
-    echo("Switched " + (if (old) "off" else "on") + " result printing.")
+    if( ! Properties.propOrFalse( "scala.repl.silentVerbositySwitch" ) )
+      echo("Switched " + (if (old) "off" else "on") + " result printing.")
```

Comment:
The property is very specific. I would have to look at the source to guess what it does.

However, `:silent -silent` is kind of cool in a weird way, to achieve the same unprinted result. Abbrev `:si -s`.

Or a `ReplProp` called `scala.repl.mute` or `warn` by analogy to log levels and `scala.repl.info`.